ulib/experimental/Steel.Effect.Atomic.fst

```diff
@@ -80,15 +80,10 @@ let ens_to_act_ens (#pre:slprop) (#a:Type) (#post:a -> slprop) (ens:fp_binary_mp
 : mprop2 pre post
 = fun m0 x m1 -> interp pre m0 /\ interp (post x) m1 /\ ens m0 x m1
 
-let atomic_repr a framed opened_invariants f pre post req ens =
+let repr a framed opened_invariants f pre post req ens =
```

Comment:
Do SteelAtomic and SteelGhost now share reprs only to avoid code duplication, or is there something more fundamental behind it?
Doing this means that Ghost computations will generate obs_at_most_one VCs at each bind/subcomp, even if it is always trivially true.

Reply:
Yes, mainly to avoid effect duplication. You are right, this would mean additional proof obligations for at most one but only when two ghost computations are composed. We can explore normalizing these proofs in the typechecker itself by making observability an exposed enum in the interface. But making the effects same removes a lot of boiler plate duplication.

Reply:
Done, no more observability related smt proofs.

Reply:
Awesome, thanks